frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "axios": "^0.23.0",
     "bootstrap": "^5.1.1",
     "delayed-scroll-restoration-polyfill": "^0.1.1",
+    "prettier": "^2.4.1",
```

Comment:
Conflictok elkerulese vegett ezt szedjuk ki innen.

Reply:
@kovarik85 értelemszerűen a package-lock.json-ből is szedjük ki.